### ISSUE TITLE
The Passion Paradox

### ISSUE THREAD

**Issue by LIYINGZHEN** [LIYINGZHEN/book-reading#66]
> Ask questions while you read—questions that you yourself must try to answer in the course of reading.

**Comment by LIYINGZHEN** [comment]
## Passion Must Be Handled with Care

- Everyone tells us to find our passion but no one tells us how to find it, let alone how to live with it. 
- While most passions start off as positive endeavors, they often take turns for the worse. 
- If you don’t proactively manage your passion, you put yourself at risk for: 		
	− Becoming a slave to external validation and results. 
	− Burnout. 
	− Regret. 
	− Loss of joy.
- If you do proactively manage your passion, however, living with passion leads to improved health, happiness, and overall life-satisfaction.
- In other words, there is both good passion and bad passion. And what direction your passion takes is largely up to you.
